### PR TITLE
i18n(es): add cli reference page

### PR DESCRIPTION
**Pull request by kevinzunigacuellar**
#### What kind of changes does this PR include?

- Translated content

#### Description

- adds `references/cli-reference.md` in Spanish

### CODE REVIEW

File: src/pages/es/reference/cli-reference.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+### `astro preview`
+
+Inicia un servidor local para servir tus archivos estáticos compilados de la carpeta `dist/`. Útil para obtener una vista previa de su sitio web usando archivos de la compilación final, antes de desplegarla.
+
+Este comando está destinado solo para pruebas locales y no está diseñado para ejecutarse en producción. Para obtener ayuda con el despliegue a producción, consulte nuestra guía sobre [implementación de un sitio web de Astro](/es/guides/deploy/).
+
+### `astro check`
+
+Ejecuta diagnósticos (como verificación de tipos dentro de archivos `.astro`) en tu proyecto y reporta errores a la consola. Si se encuentran errores, el proceso finalizará con el código **1**.
```
Comment (agustinmulet) [inline src/pages/es/reference/cli-reference.md:37]:
```suggestion
Ejecuta diagnósticos (como verificación de tipos dentro de archivos `.astro`) en tu proyecto y reporta errores en la consola. Si se encuentran errores, el proceso finalizará con el código **1**.
```

File: src/pages/es/reference/cli-reference.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+Este comando está diseñado para usarse CI workflows.
```
Comment (agustinmulet) [inline src/pages/es/reference/cli-reference.md:39]:
```suggestion
Este comando está diseñado para usarse en  workflows de CI.
```

File: src/pages/es/reference/cli-reference.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+```
+
+El comando `clear` restablece los datos de telemetría:
+
+```shell
+astro telemetry clear
+```
+
+:::tip[¿Quieres deshabilitar la telemetría en entornos CI?]
+Asegúrese de agregar el comando `astro telemetry disabled` a tus scripts de CI.
```
Comment (agustinmulet) [inline src/pages/es/reference/cli-reference.md:78]:
```suggestion
Asegúrate de agregar el comando `astro telemetry disabled` a tus scripts de CI.
```

File: src/pages/es/reference/cli-reference.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+:::
+
+## Opciones globales
+
+### `--config path`
+
+Especifica la ruta al archivo de configuración. El valor predeterminado es `astro.config.mjs`. Use esta opción cuando usas un nombre diferente en el archivo de configuración o si tienes tu archivo de configuración en otra carpeta.
```
Comment (agustinmulet) [inline src/pages/es/reference/cli-reference.md:85]:
```suggestion
Especifica la ruta al archivo de configuración. El valor predeterminado es `astro.config.mjs`. Usa esta opción si usas un nombre diferente en el archivo de configuración o si tienes tu archivo de configuración en otra carpeta.
```

File: src/pages/es/reference/cli-reference.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+astro --root myRootFolder/myProjectFolder dev
+```
+
+### `--reload`
+
+Borra el caché (las dependencias se crean dentro de las aplicaciones de Astro).
+
+### `--verbose`
+
+Habilita el registro detallado, que es útil al debugear un problema.
```
Comment (agustinmulet) [inline src/pages/es/reference/cli-reference.md:107]:
```suggestion
Habilita el registro detallado, que es útil al debuggear un problema.
```

File: src/pages/es/reference/cli-reference.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+
+### `--silent`
+
+Habilita el registro silencioso, que es útil cuando no deseas ver los registros de Astro.
+
+### `--version`
+
+Imprime el número de versión de Astro
```
Comment (agustinmulet) [inline src/pages/es/reference/cli-reference.md:115]:
```suggestion
Imprime el número de versión de Astro.
```

**Review by agustinmulet — COMMENTED** [review]
Reviewed! Good job 💪

**Comment by kevinzunigacuellar** [comment]
@Yan-Thomas LGTM!

**Comment by Yan-Thomas** [comment]
Thanks, amazing job @kevinzunigacuellar, @agustinmulet! 🙌

**Pull request merged**